pkg/daemon/daemon.go

```diff
@@ -59,7 +59,7 @@ type ptpProcess struct {
 	stopped           bool
 	logFilterRegex    string
 	cmd               *exec.Cmd
-	depProcess        []process // this could gpsd and other process which needs to be stopped if the parent process is stopped
+	depProcess        []process // this could GPSD and other process which needs to be stopped if the parent process is stopped
```

Comment:
If you're changing the comment, can you also fix the wording?  It's not really clear as is.

Reply:
done